kitsune/questions/views.py

```diff
@@ -586,7 +598,7 @@ def aaq(request, product_key=None, category_key=None, step=1, is_loginless=False
 
             if zendesk_form.is_valid() and not is_ratelimited(request, "loginless", "3/d"):
                 try:
-                    zendesk_form.send(request.user, product_config)
+                    zendesk_form.send(request.user, product)
```

Comment:
It looks like the `ZendeskForm.send()` method hasn't been updated to handle a `Product` instance instead of a product config, which I think amounts to making sure the [`ZendeskClient.create_ticket()` method can handle a `Product` instance instead of a product config](https://github.com/mozilla/kitsune/blob/5797b0c4c3457baf378834ea32651f42c0e79925/kitsune/customercare/zendesk.py#L120).